### ISSUE TITLE
Question about the re-implementation of BETA-Adv Training

### ISSUE THREAD

**Issue by ZechCal** [hyukahn16/adv_bilevel_optim#1]
Dear Author,​​

I have trained the model for 150 epochs using the following script (ensuring the learning rate decays). However, I noticed that the model still appears to suffer from robust overfitting. Do you have any guidance on reproducing your results?

python main.py --testEnabled --useBETA  --saveDir beta 

Additionally, I observed that when evaluating robustness using best_targeted_attack, the reported robustness is significantly higher than that obtained via AutoAttack. Could you clarify how the evaluation method was implemented in your paper?

I'm looking forward for your reply
Best regards

**Comment by hyukahn16** [comment]
Hi Zechcal,

Sorry for late reply. 
In this project, I tried to replicate the algorithm of the paper in hopes of reducing robust overfitting, but I wasn't able to figure out why the algorithm wasn't working. 

It's quite difficult since there's no official implementation to reference.
If you're seeking to implement the algorithm outlined in the paper, hopefully this repo can serve as a foundation but my experimental results (as you may see in the readme) were not the same as those shown in the paper.

**Comment by ZechCal** [comment]
Thank you very much. The main issue is that the method proposed in the paper contradicts my intuition—the model's fitting capacity can easily surpass the strength of the generated adversarial examples. In fact, replacing Beta with AA leads to a similar problem.
